file name encryption/AES-SIV-512-B64URL.md

```diff
@@ -0,0 +1,178 @@
+# File Name Encryption using AES-SIV-512 + Base64-URL-Encoding
+
+## Directory Metadata
+
+Every directory requires certain metadata that affects the file name encryption of its direct children:
+
+* the seed used to derive keys
+* the directory ID
+
+This data is immutable and therefore linked with a directory eternally, surviving renames/moves. This data is stored in a file called `dir.uvf`, which is stored in two places:
+1. Within the parent dir (except for root), where it serves a link to the child dir
+2. In the child dir itself (allowing disaster recovery without the parent)
+
+The exact file structure of `dir.uvf` will be discussed in more detail [below](#format-of-diruvf-and-symlinkuvf).
+
+### Directory Seed
+
+At creation time, a directory's seed is always the `latestSeed` as defined in the [vault metadata file](../vault%20metadata/README.md#sensitive-metadata). In case of the root directory this happens to also be the `initialSeed`.
+
+When navigating into a directory, the seed is read from the corresponding `dir.uvf`'s file header.
+
+> [!NOTE]
+> A directory's seed is used to derive encryption keys used to encrypt the directory's direct children. Since the seed is immutable, new child nodes added at a later time will use the same encryption keys as old children.
```

Comment:
```suggestion
> A directory's seed is used to derive file name encryption keys used to encrypt the directory's direct children's names. Since the seed is immutable, new child nodes added at a later time will use the same file name encryption keys as old children.
```